staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -2173,16 +2243,13 @@ func testPropogateStoreWithKey(ctx context.Context, t *testing.T, store *store,
 }
 
 func TestPrefix(t *testing.T) {
-	client := testserver.RunEtcd(t, nil)
-	codec := apitesting.TestCodec(codecs, examplev1.SchemeGroupVersion)
-	transformer := &prefixTransformer{prefix: []byte(defaultTestPrefix)}
 	testcases := map[string]string{
 		"custom/prefix":     "/custom/prefix",
 		"/custom//prefix//": "/custom/prefix",
 		"/registry":         "/registry",
 	}
 	for configuredPrefix, effectivePrefix := range testcases {
-		store := newStore(client, codec, nil, configuredPrefix, schema.GroupResource{Resource: "widgets"}, transformer, true, newTestLeaseManagerConfig())
```

Comment:
The `"widgets"` resource was not important to this test, and we benefit from not having one-offs in my opinion as it would mislead readers to think the distinction was necessary.

Reply:
+1